src/pages/NotificationsList/index.tsx

```diff
@@ -0,0 +1,123 @@
+// Package imports.
+import React, { useEffect, useLayoutEffect, useState } from 'react';
+import { setStatusBarBackgroundColor } from 'expo-status-bar';
+import { Text, View } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+import { Theme } from '../../constants';
+import HeaderLayout from '../../layouts/HeaderLayout';
+import { Container } from './styles';
+
+// Component imports.
+
+// Service imports.
+import notificationAPI, { NotificationType, NotificationModels } from '../../services/notifications/api';
+// Style imports.
+
+// Type declaration.
+
+// Component.
+export default function NotificationsList() : JSX.Element {
+  // Variables.
+  const navigation = useNavigation();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const [fetchedNotifications, setFetchedNotifications] = useState<any[]>([]);
```

Comment:
Does this really need to be typed as any[]?

Reply:
@labm1997 The ball is in your field.

Reply:
Fixed